tests/test_meshtastic_utils.py

```
@@ -2137,9 +2188,13 @@ def setUp(self):
 
         # Get the source module without the mock
         spec = importlib.util.find_spec("mmrelay.meshtastic_utils")
+        if spec is None:
+            raise ImportError("Could not find mmrelay.meshtastic_utils module")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)